File: Phase 2/TweetsAnalysis/src/main/java/home/playground/configs/WebMvcConfig.java
package home.playground.configs;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

@EnableWebMvc
@Configuration
@ComponentScan(basePackages = {"home.playground.resources"})
public class WebMvcConfig extends WebMvcConfigurerAdapter
{
    private static final Logger LOGGER = LoggerFactory.getLogger(WebMvcConfig.class);

    @Override
    public void addResourceHandlers(ResourceHandlerRegistry registry)
    {
        LOGGER.info("Configure static resources.");

        registry.addResourceHandler("/views/**").addResourceLocations("/WEB-INF/views/");
        registry.addResourceHandler("/resources/**").addResourceLocations("/WEB-INF/resources/");
    }
}

File: Version1-local host/ST-master/server/src/main/kotlin/home/playground/configs/WebMvcConfig.kt
package home.playground.configs

import org.slf4j.LoggerFactory
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.EnableWebMvc
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter

/**
 * Created by cuong on 11/14/15.
 */
@EnableWebMvc
@Configuration
@ComponentScan(basePackages = arrayOf("home.playground.resources"))
open class WebMvcConfig : WebMvcConfigurerAdapter()
{
    companion object {
        private val LOGGER = LoggerFactory.getLogger(WebMvcConfig::class.java)
    }

    override public fun addResourceHandlers(registry: ResourceHandlerRegistry)
    {
        LOGGER.info("Configure static resources.")

        registry.addResourceHandler("/views/**").addResourceLocations("/WEB-INF/views/")
        registry.addResourceHandler("/resources/**").addResourceLocations("/WEB-INF/resources/")
    }
}